server/middlewares/findUser/findUser.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
Please remove this and edit the eslint config

Reply:
It has been implemented